src/server/json_family.cc

```diff
@@ -84,6 +85,42 @@ string JsonType(const json& val) {
   return "";
 }
 
+bool ParseDouble(string_view src, double* value) {
```

Comment:
if you just copied the function from zset_family, I suggest to move it into common.{h,cc} below ParseHumanReadableBytes function.